src/bridge/mod.rs

```diff
@@ -36,6 +36,7 @@ use tokio_io::{AsyncRead, AsyncWrite};
 use tokio_core::reactor::Handle;
 use url::Url;
 
+
```

Comment:
I guess CI doesn't yet check formatting and the code base isn't fully rustfmt'ed anyway, but please try not to introduce arbitrary extra newlines.

Reply:
Originally missed this since my `rustfmt` runs in the background and was working as expected. After manually running it there are 6/8 invalid config options in `rustfmt.toml` and an invalid value for `fn_args_layout`.

I tried to look at modern updates for the configs but I either could not find them or they were unstable. 

I committed formatted files, but if you think this is a significant change we can discuss in an issue perhaps ?